pinot-core/src/main/java/org/apache/pinot/core/transport/ServerChannels.java

```diff
@@ -81,10 +83,11 @@ public ServerChannels(QueryRouter queryRouter, BrokerMetrics brokerMetrics, TlsC
   }
 
   public void sendRequest(String rawTableName, AsyncQueryResponse asyncQueryResponse,
-      ServerRoutingInstance serverRoutingInstance, InstanceRequest instanceRequest)
+      ServerRoutingInstance serverRoutingInstance, InstanceRequest instanceRequest, long timeoutMs)
       throws Exception {
+    byte[] requestBytes = _serializer.serialize(instanceRequest);
```

Comment:
+1

Reply:
yes, this is a very good change, I expect to find that this will significantly improve broker scalability.